release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,10 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-4-17-user-defined-network"]
+==== New UserDefinedNetwork CRD for creating primary and secondary networks (Technology Preview)
+With {product-title} {version}, users can create multiple networks and declare them as primary or secondary through the new `UserDefinedNetwork` (UDN) custom resource definition (CRD). With UDN, Users can secure traffic between namespaces without configuring and managing complex network policies. For more information, see xref:./networking/multiple_networks/will add later when docs are live.
```

Comment:
{version} doesn't replace properly. Should be {product-version} - or really I think RNs should be hard-coded because they're never really true in these instances to ever update into something other than 4.17.

Users -> users

I see your TODO, but just a reminder to replace with the proper xref. Also, I don't think we're consistent about it, but I think the for more info line would look best on its own paragraph instead of smushed into this one. But totally up to you!

```suggestion
With {product-title} {product-version}, users can create multiple networks and declare them as primary or secondary through the new `UserDefinedNetwork` (UDN) custom resource definition (CRD). With UDN, users can secure traffic between namespaces without configuring and managing complex network policies. For more information, see xref:./networking/multiple_networks/will add later when docs are live.
```